nilearn/plotting/surf_plotting.py

```diff
@@ -144,7 +144,7 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
               hemi='left', view='lateral', cmap=None,
               avg_method='mean', threshold=None, alpha='auto',
               bg_on_data=False, darkness=1, vmin=None, vmax=None,
-              title=None, output_file=None, **kwargs):
+              title=None, output_file=None, ax=None, fig=None, **kwargs):
```

Comment:
I would like to have the argument names as ax -> axes and fig -> figure. Just to be naming consistent with other volume plotting functions.